test/cli/utils_test.go

```diff
@@ -120,19 +122,9 @@ func repoRoot(tb testing.TB) string {
 func attachFileToCommandStdin(tb testing.TB, file io.Reader, command *exec.Cmd) {
 	tb.Helper()
 
-	stdin, err := command.StdinPipe()
-	if err != nil {
-		tb.Fatal(err)
-	}
-
-	_, err = io.Copy(stdin, file)
-	if err != nil {
-		tb.Fatal(err)
-	}
-	err = stdin.Close()
-	if err != nil {
-		tb.Fatal(err)
-	}
+	b, err := io.ReadAll(file)
```

Comment:
Was this erroring before? What was the reason for the change in this file?

Reply:
this approach errors when the input file is > 30k, I suspect the stdin buffer got full making the test hang for ever.